packages/strapi-database/lib/utils/primary-key.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const _ = require('lodash');
+
+module.exports = {
+  replaceIdByPrimaryKey: (params, model) => {
```

Comment:
You will need to create a test file for this new function :)

Just a styling comment: 
```js
module.exports = {
  replaceIdByPrimaryKey(params, model) {
    // code
  }
}

```